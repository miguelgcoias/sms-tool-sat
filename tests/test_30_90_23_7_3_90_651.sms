30
81 19 100 2 12 7
81 14 95 6 2 3 2 1 3 3
44 18 62 6 4 3 3 3 3 2
79 23 102 7 5 5 3 3 1 2 4
0 18 18 1 18
26 2 28 2 1 1
81 20 141 4 7 5 4 4
3 9 30 3 3 4 2
25 4 33 3 1 1 2
20 19 77 1 19
18 23 87 4 7 5 6 5
7 2 11 2 1 1
81 21 102 6 4 1 5 2 6 3
81 19 138 7 2 5 2 4 1 2 3
60 22 82 5 6 5 4 4 3
50 1 51 1 1
5 23 51 7 3 2 5 5 4 1 3
50 10 60 6 3 2 2 1 1 1
17 4 25 2 3 1
68 1 71 1 1
25 16 73 1 16
76 16 124 3 9 4 3
53 22 119 3 9 4 9
7 22 29 4 6 5 5 6
82 10 92 3 3 5 2
88 4 100 4 1 1 1 1
3 10 13 5 1 2 2 2 3
86 22 108 4 7 4 6 5
84 10 94 5 2 2 3 2 1
89 16 121 7 2 2 4 5 1 1 1
5 9 3 21 12 24
5 3 27 20 17 6
3 9 10 19
3 17 12 21
0
1 5
10 18 21 29 11 16 9 13 28 23 6
1 5
2 19 8
1 17
2 5 18
0
2 5 9
6 17 12 6 18 24 10
4 17 12 6 9
1 9
3 19 5 12
0
1 12
2 3 17
7 27 8 24 6 5 10 16
6 21 24 17 18 13 10
4 18 11 20 6
0
4 19 12 17 6
5 23 5 17 25 6
0
6 16 3 21 27 9 12
3 20 17 16
3 8 9 27
