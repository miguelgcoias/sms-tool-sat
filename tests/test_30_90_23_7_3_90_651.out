5
5 0
6 26 27
8 18 21 28
11 60 67 72 78
18 50 53 55 57 58 59
